app/src/main/java/com/dcrandroid/MainActivity.java

```diff
@@ -455,6 +455,10 @@ public void displayHistory() {
         mListView.setItemChecked(1, true);
     }
 
+    public int getPeerCount(){
```

Comment:
Use can use the `DcrConstants` class for this so it is available in other parts of the app where the instance of `MainActivity` is not available.